### PR TITLE
🧑‍💻(project) relax python version to 3.7

### PR DESCRIPTION
**Pull request by wilbrdt**
Relax required Python version for Ralph to be used as a library in older Python environment (3.7+).

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -32,7 +32,7 @@ package_dir =
     =src
 packages = find:
 zip_safe = True
-python_requires = >= 3.9
+python_requires = >= 3.7
```
Comment (jmaupetit) [inline setup.cfg:35]: Maybe also update the `Programming Language :: Python :: 3.x` classifier?

**Review by jmaupetit — COMMENTED** [review]
Thanks!

File: setup.cfg (hunk 1 above)
Comment (wilbrdt) [reply setup.cfg:35]: I guess we can, yes!

File: setup.cfg [hunk 2]
```diff
@@ -18,7 +18,7 @@ classifiers =
     License :: OSI Approved :: MIT License
     Natural Language :: English
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.7
```
Comment (jmaupetit) [inline setup.cfg:21]: You should add one line per python version:

```suggestion
    Programming Language :: Python :: 3.7
    Programming Language :: Python :: 3.8
    Programming Language :: Python :: 3.9
    Programming Language :: Python :: 3.10
    Programming Language :: Python :: 3.11
```

Comment (wilbrdt) [reply setup.cfg:21]: Whoops you're right!

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Changed
+
+- Relax required Python version to `3.7`
```
Comment (jmaupetit) [inline CHANGELOG.md:13]:
```suggestion
- Relax required Python version to `3.7`+
```

Comment (wilbrdt) [reply CHANGELOG.md]: Done!

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**